source/gwork/include/Gwork/Controls/CollapsibleList.h

```diff
@@ -61,10 +61,10 @@ namespace Gwk
             {
                 Base::List& children = GetChildren();
 
-                for (Base::List::iterator iter = children.begin(); iter != children.end(); ++iter)
+                for (auto&& iter : children)
```

Comment:
Well, with using auto I think it is a good idea to have more descriptive names to give hints about the type. `control` or `ctrl` would do since that what the child list contains. Then that is cast to a child.

Todo: rename `Base` to `Control`.